src/components/ThoughtList/Thought.jsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from "react";
+import style from './ThoughtList.css'
+
+export default function Thought({ thought, onChange, onDelete }) {
+  const [beingEdited, setBeingEdited] = useState(false);
+
+  let thoughtContent
+
+  if(beingEdited) {
+
+    thoughtContent = (
+      <div>
+        <input 
+        type="text" 
+        className={style.EditInput}
+        placeholder={thought.text}
+        onChange={(e) => {
+          onChange({
+            ...thought,
+            text: e.target.value,
+            })
+          }}
+        />
+        <button
+          type='button'
+          onClick={() => setBeingEdited(false)}
```

Comment:
on this onClick should be a editItem function does what the current onChange function is doing and setting the edit mode to false.
something like:

`const editItem(task) { handleChangeThought(task) setBeingEdited(false) };`